engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/UpdateBy.java

```diff
@@ -358,12 +358,25 @@ private void onError(@NotNull final Exception error) {
          * {@code completedAction} when the work is complete
          */
         private void computeCachedColumnRowsets(final Runnable resumeAction) {
+            // Determine which windows need to be computed.
```

Comment:
I think this code belongs in its own method now, with a more appropriate name, and called before `computeCachedColumnRowsets`.